packages/indexer/src/indexers/autocrat/autocrat-proposal-indexer.ts

```diff
@@ -276,6 +277,154 @@ export const AutocratProposalIndexer: IntervalFetchIndexer = {
             .execute()
         );
 
+        const formattedIx: TransactionInstruction = {
```

Comment:
I chose to format before inserting into the database, we could also do it when retrieving 
We could also export some functions, because the file begins to be pretty huge